website/docs/security/kyverno/creating-policy.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Creating a Simple Policy"
+sidebar_position: 133
+---
+
+To get an understanding of Kyverno Policies, we will start our lab with a simple Pod Label requirement. As you may know, Labels in Kubernetes can be used to tag objects and resources in the Cluster.
+
+Below we have a sample policy requiring a Label `CostCenter`.
+
+```file
+manifests/modules/security/kyverno/simple-policy/require-labels-policy.yaml
+```
+
+Kyverno has 2 kinds of Policy resources, **ClusterPolicy** used for Cluster-Wide Resources and **Policy** used for Namespaced Resources. The example above shows a ClusterPolicy. Take sometime to dive deep and check the below details in the configuration.
+
+* Under the spec section of the Policy, there is a an attribute `validationFailureAction` it tells Kyverno if the resource being validated should be allowed but reported `Audit` or blocked `Enforce`. Defaults to Audit, the exaple is set to Enforce.
+* The `rules` is one or more rules to be validated.
+* The `match` statement sets the scope of what will be checked. In this case, it is any `Pod` resource.
+* The `validate` statement tries to positively check what is defined. If the statement, when compared with the requested resource, is true, it is allowed. If false, it is blocked.
+* The `message` is what gets displayed to a user if this rule fails validation.
+* The `pattern` object defines what pattern will be checked in the resource. In this case, it is looking for `metadata.labels` with `CostCenter`.
+
+The Above Example Policy, will block any Pod Creation which doesn't have the label `CostCenter`.
+
+Create the policy using the following command.
+
+```bash
+$ kubectl apply -f https://raw.githubusercontent.com/aws-samples/eks-workshop-v2/main/manifests/modules/security/kyverno/simple-policy/require-labels-policy.yaml
+
+clusterpolicy.kyverno.io/require-labels created
+```
+
+Next try to create a sample Pod without any label.
+
+```bash
+$ kubectl run nginx --image=nginx:latest
```

Comment:
The ideas was to keep it as simple as possible, and not touch the rest of the workshop env. What's the idea to use the `ui` Namespace.

Reply:
The idea of using the retail sample for everything was to avoid running random container images where possible. They're yet another thing we need to maintain and it makes the labs more disjointed.